OurUmbraco/Auth/UmbracoAuthTokenDbHelper.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Umbraco.Core;
+using Umbraco.Core.Persistence;
+
+namespace OurUmbraco.Auth
+{
+    public static class UmbracoAuthTokenDbHelper
```

Comment:
Why are we using statics? Even though the codebase for our might not be that wonderful, we should refrain from making it any worse. All new code should be using DI.

Reply:
It was quick & dirty to get a POC working & was copied/pasted from super old code I had for a JWT implementation.

Will see if I can get time to update this today/tomorrow

However I upgraded the JWT nuget dependency package to latest which requires a newer version of .NET framework. I think @nul800sebastiaan said the VM running Our may not have latest frameworks available.

Reply:
It doesn't and I won't be able to install framework 4.8 either.

Reply:
For now, we're stuck on `.NET Framework 4.5.2` as confirmed through https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed

Reply:
OK thanks will see if I can fix this PR to get the JWT library to work with 4.5.2